cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -10,35 +10,46 @@
 
    Create a hero molecule.
 
-   value:           An object with the following requirements.
+   value:               An object with the following requirements.
 
-   value.heading:   A string containing title of module.
+   value.is_overlay:    A boolean to change the hero to an overlay hero.
+   value.is_white_text: A boolean to change the text color to white.
```

Comment:
The BG color is for the width of the entire header, but the image only sits within the wrapper. This is only for use when the image is dark, and only for > 600px device width. When it's less than that, it will always be black text. The approved set of colors for the entire header are all light and work with black text. @ajbush or @duelj correct me if I'm wrong.